api/v1/helper.go

```diff
@@ -37,7 +37,8 @@ const (
 	SriovCniStateAuto    = "auto"
 	SriovCniStateOff     = "off"
 	SriovCniStateOn      = "on"
-	SriovCniIpamEmpty    = "\"ipam\":{}"
+	SriovCniIpam         = "\"ipam\""
+	SriovCniIpamEmpty    = SriovCniIpam + "{}"
```

Comment:
@bn222 i believe this is whats failing unit tests ...
you need `+ ":{}"`

Reply:
ty for the catch.